test/cli/template-test-create/create/out/my-template/v0/tests/one/expected-out/.keboola/manifest.json

```diff
@@ -0,0 +1,57 @@
+{
+  "version": 2,
+  "project": {
+    "id": %d,
+    "apiHost": "%s"
```

Comment:
@webrouse Tady mám ještě jeden problém. Namísto reálnýho id a hostu a dole branch id tam potřebuju nacpat placeholdery, který potom use template command nahrazuje tady: https://github.com/keboola/keboola-as-code/blob/9e5c21dd4c6afeaa0697d0eb830d74d721aad5e4/pkg/lib/operation/template/local/test/operation.go#L114-L116
S dovolením bych to nechal už na samostatný PR. 🙂

Reply:
Jinak mám zelený testy, tak snad to bude ok. 🙂

Reply:
(to sme riesili na standupe)